src/main/kotlin/com/hedvig/underwriter/service/UnderwriterImpl.kt

```diff
@@ -31,9 +38,14 @@ import javax.money.MonetaryAmount
 class UnderwriterImpl(
     private val priceEngineService: PriceEngineService,
     private val quoteStrategyService: QuoteStrategyService,
-    private val metrics: Metrics
+    private val requotingService: RequotingService,
+    private val blockedByTypeCounter: BlockedByTypeCounter,
+    private val breachedGuidelinesCounter: BreachedGuidelinesCounter
 ) : Underwriter {
 
+    @Value("\${features.block-requoting}")
+    private var blockRequoting: Boolean = false
```

Comment:
Just a heads up I think this will always be false even if you add `features.block-requoting: true` to the config map.

I think this will do the trick:
```suggestion
    @Value("\${features.block-requoting:false}")
    lateinit var blockRequoting: Boolean
```

Reply:
`lateinit` on primitives does not work. But thanks for the tip about setting a default value in the `@Value` annotation. I am using that!

Reply:
yes forgot about that :blush: